spec/index.html

```diff
@@ -1560,7 +1570,7 @@ <h5>The <code>&lt;<dfn id="the-link-element">link</dfn>&gt;</code> element</h5>
           element does not create any links.
         </p>
         <p>
-          The <dfn id="attr-link-projection"><code>projection</code></dfn> attribute value identifies the TCRS of the linked resource, with a value from the defined set of
```

Comment:
This definition clashes with the `<map>` element's `projection` attribute definition, will have to solve this another way at some point.